cmd/whitesourceExecuteScan.go

```diff
@@ -291,9 +574,125 @@ func writeWhitesourceConfigJSON(config *ScanOptions, utils whitesourceUtils, dev
 	return nil
 }
 
+// executeNpmScan iterates over all found npm modules and performs a scan in each one.
+func executeNpmScan(config *ScanOptions, scan *whitesourceScan, utils whitesourceUtils) error {
```

Comment:
What about extracting this and related functions to an own package `pkg/whitesource/npm`?